source/clear-linux/concepts/security.rst

```diff
@@ -130,7 +129,7 @@ Compiled with secure options
 ----------------------------
 
 While |CL| packages are optimized for performance on 
-Intel Architecture, security conscious kernel and compiler options are 
+Intel® Architecture, security conscious kernel and compiler options are 
```

Comment:
I know. This is  persnickety, but it's part of our guidelines: architecture should be lowercase.